vespalog/src/logger/runserver.cpp

```diff
@@ -390,16 +390,16 @@ int main(int argc, char *argv[])
             } else {
                 fprintf(stdout, "%s was running with pid %d, sending SIGTERM\n",
                     service, pid);
-                if (killpg(pid, SIGTERM) != 0) {
+                if (kill(pid, SIGTERM) != 0) {
                     fprintf(stderr, "could not signal %d: %s\n", pid,
                             strerror(errno));
                     return 1;
                 }
             }
-            fprintf(stdout, "Waiting for exit (up to 60 seconds)\n");
-            for (int cnt(0); cnt < 1800; cnt++) {
+            fprintf(stdout, "Waiting for exit (up to 15 minutes)\n");
```

Comment:
@arnej27959 and @baldersheim: If we are not able to differentiate between processes, perhaps we should be conservative at first and increase this to only 120 seconds?